### PR TITLE
test: Add test for vertices and edge count in pgraph

### PR DESCRIPTION
**Pull request by jhu960213**
Added a test for vertices and edge count in pgraph.

### CODE REVIEW

File: pgraph/pgraph_test.go [hunk 1]
```diff
@@ -49,6 +51,54 @@ func TestCount1(t *testing.T) {
...
+	// populate vertices
+	var vertices [6]Vertex
+	for i := 0; i < len(vertices); i++ {
+		var name string = "v" + strconv.Itoa(i+1)
+		vertices[i] = NV(name)
+	}
+
+	// populate edges
+	var edges [5]Edge
+	for x := 0; x < len(edges); x++ {
```
Comment (purpleidea) [inline pgraph/pgraph_test.go:75]: You can re-use `i` as a counter variable here and below...

File: pgraph/pgraph_test.go [hunk 2]
```diff
@@ -49,6 +51,54 @@ func TestCount1(t *testing.T) {
...
+		var name string = "e" + strconv.Itoa(x+1)
```
Comment (purpleidea) [inline pgraph/pgraph_test.go:76]: name := ...

File: pgraph/pgraph_test.go [hunk 3]
```diff
@@ -49,6 +51,54 @@ func TestCount1(t *testing.T) {
...
+	}
+
+	// add all the vertices
+	for a := 0; a < len(vertices); a++ {
+		G.AddVertex(vertices[a])
+	}
+
+	// add edges to the vertices
+	for a := 0; a < len(vertices); a++ {
+		var b int = a + 1
```
Comment (purpleidea) [inline pgraph/pgraph_test.go:87]: b := a + 1

File: pgraph/pgraph_test.go [hunk 4]
```diff
@@ -49,6 +51,54 @@ func TestCount1(t *testing.T) {
...
+		if a <= len(vertices)-2 {
```
Comment (purpleidea) [inline pgraph/pgraph_test.go:88]: Subtle: -2's don't usually appear in code as much as -1's. Since we're a zero-based language, it's expected to see a -1 here and there... But you're using the -2 off of the first of the two elements... So it raises an eyebrow... So probably best to use -1 and compare it to b instead.

**Review by purpleidea — COMMENTED** [review]
Thanks for the patch! As I hear that you're learning golang, I decided to nit a few things to help you practice your rebasing. Let me know if it makes sense or if you have any questions.

Thanks!

**Comment by purpleidea** [comment]
Conflicts so closing for now.
